src/styles/blocks/card.scss

```
@@ -0,0 +1,78 @@
+@import "../utils/variables";
+@import "../utils/extends";
+
+.card {
+  &__image {
+    @extend %imac-size;
+  }
+  &__title {
```

```suggestion
  }
  
  &__title {
```
add empty lines between blocks